client/src/graphs/svg/renderers/EpiRenderer.ts

```diff
@@ -196,15 +196,12 @@ export default class EpiRenderer extends SVGRenderer {
       .style('stroke-width', DEFAULT_STYLE.node.strokeWidth + 3);
   }
 
-  highlightSubgraph (subgraph: SubgraphInterface): void {
+  highlightNodes (nodesList: SubgraphNodeInterface[]): void {
     const chart = (this as any).chart;
-    const nodes = subgraph.nodes;
-
     chart.selectAll('.node-ui').each(function (d) {
-      const isHighlighted = nodes.map(node => node.id).includes(d.id);
+      const isHighlighted = nodesList.map(node => node.id).includes(d.id);
       d3.select(this).select('rect')
-        .style('stroke', isHighlighted ? Colors.HIGHLIGHT : Colors.STROKE)
-        .style('stroke-width', DEFAULT_STYLE.node.strokeWidth + (isHighlighted ? 3 : 0));
+        .style('fill', isHighlighted ? Colors.NODES.EDITED : EpiRenderer.calcNodeColor(d));
```

Comment:
Did you mean to revert back to node fill highlighting? Now the simulate page looks like this.

<img width="1131" alt="Screen Shot 2021-07-20 at 9 20 15 AM" src="https://user-images.githubusercontent.com/14062458/126331201-ecda61cc-3748-4d07-870f-18870659a938.png">

Reply:
![image](https://user-images.githubusercontent.com/10552785/126334235-f2c1f3fd-6e20-4d2e-8b80-e01c8d10c7bb.png)
This is how it looks for me @mj3cheun

Reply:
So yes, this was intentional as discussed over slack.